build.psm1

```diff
@@ -1139,7 +1146,8 @@ function Start-PSPester {
     }
 
     # Autoload (in subprocess) temporary modules used in our tests
-    $command += '$env:PSModulePath = '+"'$TestModulePath$TestModulePathSeparator'" + '+$($env:PSModulePath);'
+    $newPathFragment = $TestModulePath + $TestModulePathSeparator + $script:TestModuleDirPath + $TestModulePathSeparator
```

Comment:
`$TestModulePath` and `TestModuleDirPath` look a bit confusing. It would be nice if we can pull the `SelfSignedCertificate` module to the `test\tools\Modules` folder, and update `.gitignore` to ignore files in the `SelfSignedCertificate` folder. Only update/overwrite it in `Start-PSPester` when we bump the version.

This is not a blocking comment for this PR, but it would be great if it can be addressed in a follow-up PR.

Reply:
@daxian-dbw polluting the git directories is counter-productive to the work to enable the tests to run outside of the git repo

Reply:
See https://github.com/PowerShell/PowerShell/pull/7875#discussion_r220750974 and https://github.com/PowerShell/PowerShell/pull/7875#pullrequestreview-159232521.

Reply:
Good to know that thoughts have been put on this issue. Closed.